--- CMakeLists.txt	1970-01-01 10:00:00.000000000 +1000
+++ CMakeLists.txt	2016-04-29 14:35:41.658557800 +1000
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8)
+project(LIBCACA)
+
+set(LIBCACA_SRC caca/driver/conio.c caca/driver/gl.c 
+    caca/driver/ncurses.c caca/driver/null.c 
+    caca/driver/raw.c caca/driver/slang.c 
+    caca/driver/vga.c caca/driver/win32.c
+    caca/driver/x11.c caca/codec/export.c 
+    caca/codec/import.c caca/codec/text.c
+    caca/attr.c caca/box.c caca/caca.c 
+    caca/caca_conio.c caca/canvas.c caca/charset.c
+    caca/conic.c caca/dirty.c caca/dither.c caca/event.c 
+    caca/figfont.c caca/file.c caca/font.c 
+    caca/frame.c caca/getopt.c caca/graphics.c 
+    caca/legacy.c caca/line.c caca/prof.c caca/string.c 
+    caca/time.c caca/transform.c caca/triangle.c)
+    
+add_library (libcaca SHARED ${LIBCACA_SRC})
+add_definitions("-D_WIN32")
+add_definitions("-DNDEBUG")
+add_definitions("-D_LIB")
+add_definitions("-D__LIBCACA__")
+add_definitions("-DDLL_EXPORT")
+add_definitions("-D_CRT_SECURE_NO_WARNINGS")
+include_directories(caca caca/codec win32)
+
+install(TARGETS libcaca 
+  RUNTIME DESTINATION bin
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib)
+install(FILES caca/caca.h caca/caca0.h caca/caca_conio.h caca/caca_types.h DESTINATION include)  
