--- makedef.py.empty	2016-01-28 09:02:16.497564700 +1000
+++ makedef.py	2016-01-28 08:48:23.788580400 +1000
@@ -0,0 +1,15 @@
+
+import sys
+expfile = sys.argv[1]
+deffile = sys.argv[2]
+
+fh = open(deffile, 'w')
+fh.write('EXPORTS\n')
+
+for line in open(expfile):
+    line = line.strip('\r\n')
+    arr = line.split()
+    if len(arr) == 2 and line.startswith(' ') and arr[1].startswith('?'):
+        fh.write('    %s\n' % arr[1])
+    
+fh.close()
