--- laszip\src\mydefs .hpp.orig	2015-08-17 07:35:36.000000000 +1000
+++ LASzip\src\mydefs.hpp	2015-12-22 13:26:24.069502100 +1000
@@ -53,7 +53,7 @@
 typedef float              F32;
 typedef double             F64;
 
-#if defined(_MSC_VER) && (_MSC_VER < 1300) || defined (__MINGW32__)
+#if defined(_MSC_VER) || defined (__MINGW32__)
 typedef int                BOOL;
 #else
 typedef bool               BOOL;
