--- makefile.vc.empty	2016-01-26 17:15:21.879226600 +1000
+++ makefile.vc	2016-01-27 18:04:36.812359900 +1000
@@ -0,0 +1,52 @@
+
+CPPFLAGS = /Ox /EHsc /ILASlib\inc /ILASzip\src /D_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS /DNDEBUG /MD /nologo
+
+OBJ_LAS		= LASlib\src\lasreader.obj LASlib\src\laswriter.obj LASlib\src\lasreader_las.obj LASlib\src\lasreader_bin.obj LASlib\src\lasreader_qfit.obj LASlib\src\lasreader_shp.obj LASlib\src\lasreader_asc.obj LASlib\src\lasreader_bil.obj LASlib\src\lasreader_dtm.obj LASlib\src\lasreader_txt.obj LASlib\src\lasreadermerged.obj LASlib\src\lasreaderbuffered.obj LASlib\src\lasreaderpipeon.obj LASlib\src\laswriter_las.obj LASlib\src\laswriter_bin.obj LASlib\src\laswriter_qfit.obj LASlib\src\laswriter_wrl.obj LASlib\src\laswriter_txt.obj LASlib\src\laswritercompatible.obj LASlib\src\laswaveform13reader.obj LASlib\src\laswaveform13writer.obj LASlib\src\lasutility.obj LASlib\src\lasfilter.obj LASlib\src\lastransform.obj LASlib\src\fopen_compressed.obj
+OBJ_LAZ		= LASzip\src\laszip.obj LASzip\src\lasreadpoint.obj LASzip\src\lasreaditemcompressed_v1.obj LASzip\src\lasreaditemcompressed_v2.obj LASzip\src\laswritepoint.obj  LASzip\src\laswriteitemcompressed_v1.obj LASzip\src\laswriteitemcompressed_v2.obj LASzip\src\integercompressor.obj LASzip\src\arithmeticdecoder.obj LASzip\src\arithmeticencoder.obj LASzip\src\arithmeticmodel.obj LASzip\src\lasindex.obj LASzip\src\lasquadtree.obj LASzip\src\lasinterval.obj 
+
+.cpp.obj:
+  $(CPP) $(CPPFLAGS) /Fo$*.obj /c $<
+
+all: laslib.dll laszip.exe lasinfo.exe lasprecision.exe las2txt.exe txt2las.exe las2las.exe lasdiff.exe lasmerge.exe lasindex.exe
+  
+laslib_s.lib: $(OBJ_LAZ) $(OBJ_LAS)
+    lib.exe $(OBJ_LAZ) $(OBJ_LAS) /OUT:laslib_s.lib
+    
+laslib_s.exp: laslib_s.lib
+    dumpbin.exe laslib_s.lib /LINKERMEMBER:1 /OUT:laslib_s.exp
+    
+laslib.def: laslib_s.exp
+    python makedef.py laslib_s.exp laslib.def
+    
+laslib.dll: laslib_s.lib laslib.def
+    link.exe /DLL /OUT:laslib.dll /DEF:laslib.def laslib_s.lib
+    
+laszip.exe: src\laszip.obj src\geoprojectionconverter.obj
+    link.exe src\laszip.obj src\geoprojectionconverter.obj laslib.lib /OUT:laszip.exe 
+
+lasinfo.exe: src\lasinfo.obj src\geoprojectionconverter.obj
+    link.exe src\lasinfo.obj src\geoprojectionconverter.obj laslib.lib /OUT:lasinfo.exe 
+
+lasprecision.exe: src\lasprecision.obj src\geoprojectionconverter.obj
+    link.exe src\lasprecision.obj src\geoprojectionconverter.obj laslib.lib /OUT:lasprecision.exe 
+
+las2txt.exe: src\las2txt.obj
+    link.exe src\las2txt.obj laslib.lib /OUT:las2txt.exe 
+
+txt2las.exe: src\txt2las.obj src\geoprojectionconverter.obj
+    link.exe src\txt2las.obj src\geoprojectionconverter.obj laslib.lib /OUT:txt2las.exe 
+
+las2las.exe: src\las2las.obj src\geoprojectionconverter.obj
+    link.exe src\las2las.obj src\geoprojectionconverter.obj laslib.lib /OUT:las2las.exe 
+
+lasdiff.exe: src\lasdiff.obj
+    link.exe src\lasdiff.obj laslib.lib /OUT:lasdiff.exe 
+
+lasmerge.exe: src\lasmerge.obj src\geoprojectionconverter.obj
+    link.exe src\lasmerge.obj src\geoprojectionconverter.obj laslib.lib /OUT:lasmerge.exe 
+
+lasindex.exe: src\lasindex.obj
+    link.exe src\lasindex.obj laslib.lib /OUT:lasindex.exe 
+    
+clean:
+    del $(OBJ_LAZ) $(OBJ_LAS) *.exp *.def *.lib *.exe src\*.obj
